tests/test_iib_client.py

```diff
@@ -202,6 +202,28 @@ def fixture_create_empty_index_build_details_json():
     return json
 
 
+@pytest.fixture
+def fixture_recursive_related_bundles_build_details_json():
+    json = {
+        "id": 3,
```

Comment:
Minor: It would be better to have the id as 5, as the build is registered with id 5. It won't impact any functionality but it's nice to have consistency. It's not major enough to hold off the PR

Reply:
P.S tox tests are failing

Reply:
Thanks, I edited also the id in fixture_create_empty_index_build_details_json pytest.fixture. Please check.